<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-4P9SU403" failures="13" tests="235" name="runner.TestRunner" time="224.949" errors="1" timestamp="2023-08-15T14:26:08 CDT" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="1.162"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.260"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.508"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.668"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.891"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.643">
    <failure message="expected [password can’t be too similar to your other personal information.] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password can’t be too similar to your other personal information.] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.RegisterPageSD.it_should_display_an_error_message(RegisterPageSD.java:92)
at ✽.It should display an error message "password can’t be too similar to your other personal information."(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:80)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.622"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.846"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.442"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.944"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.871"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.077">
    <failure message="Title do not match expected [NumpyNinja] but found [Registration]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Title do not match expected [NumpyNinja] but found [Registration]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at stepDefinitions.RegisterPageSD.the_user_should_be_redirected_to_homepage_with_the_message(RegisterPageSD.java:214)
at ✽.The user should be redirected to Homepage with the message "New Account Created. You are logged in as Numpy@sdet117_166"(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:98)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.658"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.910"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.199"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.330"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.414"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.545"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.720"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.711"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.374"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.868"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.781"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.489"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.958"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.537"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.231"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.850"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.173"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.641"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.422"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.937"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.121"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.906"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.966"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.684"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.464"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.620"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.162"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.884"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.027"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.454"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.543"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.170"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.222">
    <failure message="expected [Password should contain at least 8 characters] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Password should contain at least 8 characters] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.RegisterPageSD.it_should_display_an_error_message(RegisterPageSD.java:92)
at ✽.It should display an error message "Password should contain at least 8 characters"(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:64)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="1.253"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.964"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.211"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.079"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.352"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.142"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.462"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.469"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.435"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.585"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.595">
    <failure message="expected [Password can’t be commonly used password] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Password can’t be commonly used password] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.RegisterPageSD.it_should_display_an_error_message(RegisterPageSD.java:92)
at ✽.It should display an error message "Password can’t be commonly used password"(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:89)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.877"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.233"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.883"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.535"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.524"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.417"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.513"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.180"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.936"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.018"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.457"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.350"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.030"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.088"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.627"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.185"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.169"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.520"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.573"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.208"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.284"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.119"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.871"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.868"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.589">
    <failure message="expected [Username already exists] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Username already exists] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.RegisterPageSD.it_should_display_an_error_message(RegisterPageSD.java:92)
at ✽.It should display an error message "Username already exists"(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:48)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.943"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.599"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.190"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.504"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.922"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.217"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.140"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.832"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.606"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.001"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.429"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.554"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.935"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.421"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.277"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.223">
    <failure message="expected [Password can’t be entirely numeric.] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Password can’t be entirely numeric.] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.RegisterPageSD.it_should_display_an_error_message(RegisterPageSD.java:92)
at ✽.It should display an error message "Password can’t be entirely numeric."(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:72)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.451"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.619"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.848">
    <failure message="expected [Invalid Username] but found [Invalid Username and Password]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Invalid Username] but found [Invalid Username and Password]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.SigninPageSD.click_login_button(SigninPageSD.java:125)
at ✽.click login button(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/3_SigninPage.feature:37)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.302"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.155"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.766">
    <failure message="Result do not match expected [Submission Successful] but found [ No tests were collected]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [Submission Successful] but found [ No tests were collected]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at stepDefinitions.ArraySD.the_user_should_be_presented_with_successful_submission_message(ArraySD.java:167)
at ✽.The user should be presented with successful submission message(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/5_Array.feature:357)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.532"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.653"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.226"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.303"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.479">
    <failure message="expected [Invalid Password] but found [Invalid Username and Password]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Invalid Password] but found [Invalid Username and Password]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.SigninPageSD.click_login_button(SigninPageSD.java:125)
at ✽.click login button(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/3_SigninPage.feature:37)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.252"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.500">
    <failure message="Result do not match expected [Submission Successful] but found [Error occurred during submission]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [Submission Successful] but found [Error occurred during submission]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at stepDefinitions.ArraySD.the_user_should_be_presented_with_successful_submission_message(ArraySD.java:167)
at ✽.The user should be presented with successful submission message(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/5_Array.feature:265)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.147"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.276"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.540"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.706"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.671"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.656"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.528"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.253"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.600"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.759"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.816"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.377"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.132"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.637"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.312"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.587"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.670"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.533"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.876"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.585"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.178"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.915"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.204"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.137"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.402"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.445">
    <failure message="Result do not match expected [Submission Successful] but found [Error occurred during submission]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [Submission Successful] but found [Error occurred during submission]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at stepDefinitions.ArraySD.the_user_should_be_presented_with_successful_submission_message(ArraySD.java:167)
at ✽.The user should be presented with successful submission message(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/5_Array.feature:310)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.590"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.082"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.712"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.956"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.456"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.483"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.954"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.077"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.453"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.665"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.544"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.607"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.933"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.681"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.139"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.081"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.378"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.139"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.550"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.483"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.438"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.039"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.525"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.513"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.694"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.772"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.849"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.477"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.572"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.511"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.441"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.337"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.544"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.268"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.135"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.051"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.402"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.627"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.169"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.557"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.877"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.910"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.689"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.468"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.643"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.166"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.253"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.828"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.038"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.389"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.239"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.077"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.868"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.906"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.946"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.461">
    <error message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//a[@href=&#039;/logout&#039;]&quot;}
  (Session info: chrome=115.0.5790.171)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &#039;4.10.0&#039;, revision: &#039;c14d967899&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;20.0.1&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [19ac61f1522e865f708fca636e58d2f0, findElement {using=xpath, value=//a[@href=&#039;/logout&#039;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 115.0.5790.171, chrome: {chromedriverVersion: 114.0.5735.90 (386bc09e8f4f..., userDataDir: C:\Users\14793\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:53855}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:53855/devtoo..., se:cdpVersion: 115.0.5790.171, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 19ac61f1522e865f708fca636e58d2f0" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//a[@href='/logout']"}
  (Session info: chrome=115.0.5790.171)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '20.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [19ac61f1522e865f708fca636e58d2f0, findElement {using=xpath, value=//a[@href='/logout']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 115.0.5790.171, chrome: {chromedriverVersion: 114.0.5735.90 (386bc09e8f4f..., userDataDir: C:\Users\14793\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:53855}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:53855/devtoo..., se:cdpVersion: 115.0.5790.171, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 19ac61f1522e865f708fca636e58d2f0
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:199)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:132)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:51)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:531)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:350)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy60.click(Unknown Source)
at pageObjects.RegisterPage.clickSignOutLink(RegisterPage.java:182)
at stepDefinitions.RegisterPageSD.the_user_clicks_the_link(RegisterPageSD.java:226)
at ✽.The user clicks the "Sign Out" link(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:104)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="1.268"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.813"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.277"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.942"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.237"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.488"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.737"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.325"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.967">
    <failure message="expected [Please enter a valid username] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Please enter a valid username] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.RegisterPageSD.it_should_display_an_error_message(RegisterPageSD.java:92)
at ✽.It should display an error message "Please enter a valid username"(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/2_RegisterPage.feature:40)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="1.012"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.554"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.547"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.355"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.008"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.489"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.197"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.895"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.891"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.683"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.109"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.923"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.547"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.544"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.431"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.355"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.178"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.885"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.587"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.866"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.354"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.701"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.177"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.480"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="2.294">
    <failure message="Result do not match expected [Submission Successful] but found [Error occurred during submission]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [Submission Successful] but found [Error occurred during submission]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at stepDefinitions.ArraySD.the_user_should_be_presented_with_successful_submission_message(ArraySD.java:167)
at ✽.The user should be presented with successful submission message(file:///C:/Users/14793/eclipse-workspace/DSAlgosample/DSAlgo/src/test/resources/features/5_Array.feature:219)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="0.888"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.371"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.603"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="1.316"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.540"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.624"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.458"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.552"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="0.512"/>
</testsuite> <!-- runner.TestRunner -->
